script.js

```diff
@@ -0,0 +1,18 @@
+function findElement(text) {
+    const removeChars = text.replace(/[^a-zA-Z0-9 ]/g,"")
+    const trimmedStr = removeChars.replace(/\s+/g, " ").trim()
+    const styleArr = trimmedStr.split(' ')
+    const element = styleArr[0]
+    const attr = styleArr[1]
+    const value = styleArr[2]
+    $(document).find(element).css({'color': value})
+}
```

Comment:
Descriptive function names and splitting this up into multiple functions would help with clarity. Perhaps `parseInput` and `changeStyle` or something. And some comments to say what each step is doing. Good job creating a parser for your text though! That's pretty cool. :)